Improve "Foreman on AWS" guide (#3485)
theforeman/foreman-documentation: guides/common/modules/con_unsupported-use-cases-for-foreman-on-aws.adoc

[id="unsupported-use-cases-for-{project-context}-on-aws"]
= Unsupported use cases for {Project} on AWS

In Amazon Web Services (AWS), you cannot manage the DHCP service.

> Because of this, most of Kickstart and PXE provisioning models of {Project} are unusable.

Reviewer: ```suggestion
Because of this, you cannot use most of Kickstart and PXE provisioning models of {Project}.
```

[id="unsupported-use-cases-for-{project-context}-on-aws"]
= Unsupported use cases for {Project} on AWS

> In AWS, you cannot manage the DHCP.

Reviewer 2: Note that if you add a Smart Proxy using a Smart Proxy in our on-premise data center (see Scenario 2: Connecting on-premises and AWS region) then you can use these methods.

This is a long standing issue I have with this guide.

Author: I consider this out of scope for now. I am not that deep into AWS to test this. I can create a follow-up issue to make this visible.
